src/Java.Interop.Tools.JavaCallableWrappers/Java.Interop.Tools.JavaCallableWrappers/JavaCallableWrapperGenerator.cs

```diff
@@ -95,36 +64,39 @@ public string GetJavaAccess ()
 		List<Signature> methods = new List<Signature> ();
 		List<Signature> ctors   = new List<Signature> ();
 		List<JavaCallableWrapperGenerator> children;
-		List<OverriddenMethodDescriptor> overriddenMethodDescriptors;
+
 		readonly IMetadataResolver cache;
+		readonly JavaCallableMethodClassifier methodClassifier;
 
 		[Obsolete ("Use the TypeDefinitionCache overload for better performance.")]
 		public JavaCallableWrapperGenerator (TypeDefinition type, Action<string, object []> log)
 			: this (type, null, log, resolver: null)
 		{ }
 
-		public JavaCallableWrapperGenerator (TypeDefinition type, Action<string, object[]> log, TypeDefinitionCache cache)
```

Comment:
This is an ABI break.

The question is whether or not this is an *important* API break, i.e. do we need to maintain ABI for this member?  `Xamarin.Android.Build.Tasks.dll` references this member, so removing this member could cause "debugging complications" if you wanted to e.g. use a *newer* `Java.Interop.Tools.JavaCallableWrappers.dll` with an *older* `Xamarin.Android.Build.Tasks.dll`.  That's not a "supported" scenario, certainly, but is it a scenario we care about anyway?

I don't know.